### PR TITLE
zh-CN: update for 'using-gc-traces.md'

### PR DESCRIPTION
**Pull request by ghost**
Refs: https://github.com/nodejs/nodejs.org/pull/4775.

### CODE REVIEW

**Comment by ghost** [comment]
@willin ： Can you help to have a review if free? thanks!

**Comment by tony-go** [comment]
Awesome 🙌

File: locale/zh-cn/docs/guides/diagnostics/memory/using-gc-traces.md [hunk 1]
```diff
@@ -5,100 +5,282 @@ layout: docs.hbs
 
 # 追踪垃圾回收
 
+本指南将贯穿所有关于垃圾回收跟踪机制的基础部分。
+
+当你完成本章节阅读之后，你将会：
```
Comment (willin) [inline locale/zh-cn/docs/guides/diagnostics/memory/using-gc-traces.md]: how about `你将会了解：`

**Review by willin — APPROVED** [review]
LGTM

File: locale/zh-cn/docs/guides/diagnostics/memory/using-gc-traces.md (hunk 1 above)
Comment (ghost) [reply locale/zh-cn/docs/guides/diagnostics/memory/using-gc-traces.md]: Thanks！

**Review by willin — APPROVED** [review]

**Pull request merged**